src/frontend/next/src/components/MyFeeds.tsx

```diff
@@ -55,7 +55,7 @@ const MyFeeds = ({ user }: MyFeedsProps) => {
         const userFeeds = await response.json();
 
         const userFeedHash = userFeeds.reduce((hash: FeedHash, feed: Feed) => {
-          hash[feed.id] = { author: feed.author, url: feed.url };
+          hash[feed.id] = { author: feed.author, url: feed.url, link: feed.link };
```

Comment:
We don't need `url: feed.url` anymore, right? maybe we can delete it.

Reply:
@manekenpix I thought so too, but the hash is then passed to `ExistingFeedList` where the RSS feed URL is needed.